src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1011,6 +1014,8 @@ MulticopterPositionControl::start_flight_task()
 			// No task was activated.
 			_flight_tasks.switchTask(FlightTaskIndex::None);
 		}
+	} else if (should_disable_task) {
```

Comment:
@bkueng I was wondering if we could somehow do the same thing for the case when we are a vtol in fixed wing mode. Currently in that case we are just returning at the top of this function. I don't have a good idea though.

Reply:
I noticed that too. The problem there is that I don't want to continuously switch between a task an None.